test/hotspot/jtreg/vmTestbase/gc/gctests/mallocWithGC2/mallocWithGC2.java

```diff
@@ -113,8 +113,6 @@ public void run() {
                                 tArray[i].start();
 
                         tArray[0].join(); // wait for the javaHeapEater Thread to finish
-                        tArray[1].stop(); // Once javaHeapEater is finished, stop the
```

Comment:
So without this the other thread will run for a full 3 minutes - is that a concern?

Reply:
VM doesn't wait for the completion of threads that are executed native code. Seems like a new bug?

Reply:
@lmesnik I don't understand what you mean here. The cHeapEater thread is a non-daemon thread so the test program won't terminate until it does. In the old code we would assist it to terminate early using stop() but in the new code it will run until it completes normally after 180 seconds.